alex/applications/PublicTransportInfoCS/data/convert_idos_stops.py

```diff
@@ -28,7 +28,7 @@
 import sys
 import re
 from add_cities_to_stops import load_list, get_city_for_stop
-from alex.utils.various import remove_dups_stable
+from alex.utils.various import list_remove_duplicates_keep_ordering
```

Comment:
Oh, I've just realized that I wrote this function... maybe that's why it seemed so clear to me :smile: . So I'm probably not the right one to judge... `remove_list_dups_stable` would sound slightly better to me, but i'm OK with both.